src/game/version.h

```diff
@@ -5,5 +5,6 @@
 #include <generated/nethash.cpp>
 #define GAME_VERSION "0.7 trunk"
 #define GAME_NETVERSION "0.7 " GAME_NETVERSION_HASH
+#define GAME_NETGUI_VERSION "0.7-0.1.b20151228"
```

Comment:
NetGUI will be no more compatible. All IDs NetMsg are different (I guess, since it's probably based on the order). For now, this line is not needed